doc/sal_constraints_and_recommendations.rst

```diff
@@ -561,6 +595,8 @@ An easy way to tell if a particular unit is valid is to try to make an astropy.u
     These units may not make sense but are valid according to the astropy parser.
     Please note this "feature" of astropy. 
```

Comment:
It would be helpful to say that these units are compatible with Astropy.

And I strongly suggest replacing this current comment with that information. The current comment seems needlessly snarky. What "feature" are you referring to?

Reply:
This comment, which I didn't write, refers to astropy excepting units such "deg22" which apparently is a degree in the 22nd power. Apparently it is a nonsensical unit based on the comment and so people should be wary of the feature according to the comment.

Reply:
I would appreciate it.